platform/ios/Integration Tests/Snapshotter Tests/MGLMapSnapshotterTest.m

```diff
@@ -106,20 +108,29 @@ - (void)testDeallocatingSnapshotterDuringSnapshot🔒 {
 //  dispatch_queue_t backgroundQueue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
 
     __weak __typeof__(self) weakself = self;
+    __block __weak MGLMapSnapshotter *weakSnapshotter;
 
     dispatch_async(backgroundQueue, ^{
 
         dispatch_group_t dg = dispatch_group_create();
         dispatch_group_enter(dg);
 
         dispatch_async(dispatch_get_main_queue(), ^{
-            MGLMapSnapshotter *snapshotter = snapshotterWithCoordinates(coord, size);
-            [snapshotter startWithCompletionHandler:^(MGLMapSnapshot * _Nullable snapshot, NSError * _Nullable error) {
-                // This completion block should not be called.
-                __typeof__(self) strongself = weakself;
-                MGLTestFail(strongself, @"Completion handler should not be called after snapshotter is deallocated.");
-            }];
-            dispatch_group_leave(dg);
+            @autoreleasepool {
+                MGLMapSnapshotter *snapshotter = snapshotterWithCoordinates(coord, size);
+                weakSnapshotter = snapshotter;
+                
+                [snapshotter startWithCompletionHandler:^(MGLMapSnapshot * _Nullable snapshot, NSError * _Nullable error) {
+                    // We expect this completion block to be called with an error
```

Comment:
This comment is inaccurate; it crept back in when I reverted #210’s changes to the test, but we want the completion handler to be called with a valid snapshot now.